src/stores/asynchronousAction.ts

```diff
@@ -7,6 +7,11 @@ export interface IAsynchronousDispatch<S>
   <R>(action: IAsynchronousAction<S, R>): Promise<R>;
 }
 
+/**
+ * Refactor to support template string types once
+ * TypeScript 4.1 is released.
+ * https://github.com/microsoft/TypeScript/pull/40336
+ */
 export interface IActionTypeCollection {
```

Comment:
Once TS 4.1 is released, we would be able to get auto-completion for action types (e.g. `types.request` will be `ACTION_TYPE_REQUEST`)